src/main/java/com/blockout/controls/AbstractTextElement.java

```diff
@@ -12,12 +12,13 @@
 {
     protected static final ResourceLocation TEXTURE       = new ResourceLocation("textures/gui/widgets.png");
     protected              double           scale         = 1.0;
-    protected              Alignment        textAlignment = Alignment.MiddleLeft;
+    protected              Alignment        textAlignment = Alignment.MIDDLE_LEFT;
     protected              int              textColor     = 0xffffff;
     protected              boolean          shadow        = false;
 
     public AbstractTextElement()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)